### PR TITLE
feat(dedicated.server): disable firewall and backup section in US

### PR DESCRIPTION
**Pull request by Jisay**
ref to come :-)

### CODE REVIEW

File: client/app/dedicated/server/dedicated-server.html [hunk 1]
```diff
@@ -27,13 +27,15 @@
                                   data-disabled="server.isExpired"></oui-header-tabs-item>
             <oui-header-tabs-item data-text="{{:: 'server_tab_ftp_backup' | translate }}"
                                   data-state="app.dedicated.server.ftpBackup"
-                                  data-disabled="server.isExpired"></oui-header-tabs-item>
+                                  data-disabled="server.isExpired"
+                                  data-ng-if="worldPart !== 'US'"></oui-header-tabs-item>
```
Comment (marie-j) [inline client/app/dedicated/server/dedicated-server.html]: we should probably uniformize with `coreConfig.getRegion()`

File: client/app/dedicated/server/firewall/dedicated-server-firewall.routes.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+        'tabView@app.dedicated.server': {
+          template,
+        },
       },
-    },
-    translations: { value: ['..'], format: 'json' },
-  });
+      translations: { value: ['..'], format: 'json' },
+    });
+  }
```
Comment (marie-j) [inline client/app/dedicated/server/firewall/dedicated-server-firewall.routes.js]: Don't know if it's better to have a `redirectTo` to handle this that will redirect to `app.dedicated.server` for US instead

**Comment by jleveugle** [comment]
Hello @Jisay,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

File: client/app/dedicated/server/dedicated-server.html (hunk 1 above)
Comment (Jisay) [reply client/app/dedicated/server/dedicated-server.html]: This is the case in the associated controller:
https://github.com/ovh-ux/ovh-manager-dedicated/blob/7ec657111e2abc5aaa064e580d0c33abe93965df/client/app/dedicated/server/dedicated-server.controller.js#L42

**Comment by antleblanc** [comment]
@Jisay Is there any internal ticket reference related to this Pull request?

Thanks!

**Pull request merged**